experimental/dds/tree2/src/test/utils.ts

```diff
@@ -848,3 +843,13 @@ export function defaultRevisionMetadataFromChanges(
 	}
 	return revisionMetadataSourceFromInfo(revInfos);
 }
+
+/**
+ * Helper for building {@link NamedTreeSchema} without using {@link SchemaBuilder}.
+ */
+export function namedTreeSchema(data: Partial<TreeSchemaBuilder> & Named<string>): NamedTreeSchema {
```

Comment:
Is the need for a helper a possible indication that `SchemaBuilder` needs a refactor?
If so it would likely not be tackled in this PR so feel free to resolve (or leave unresolved) without further action.